src/cleed++/Core/Basis.hh

```
@@ -0,0 +1,74 @@
+    Basis& setA3(double a3_x, double a3_y, double a3_z);
+    Basis& setBasis(const Coordinate &a1, const Coordinate &a2, const Coordinate &a3);
+    Basis& setBasis(const Basis &basis);
+    Basis& setBasis(const Basis *basis);
+
+    /* getters */
+    const Coordinate getA1() const;
+    const Coordinate getA2() const;
+    const Coordinate getA3() const;
+
+    /* other methods */
+    double **calculateRotationMatrix();
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)